packages/components/src/components/scroller/scroller-styles.ts

```diff
@@ -81,20 +81,22 @@ export const getComponentCss = (
         border: 0,
         outline: 0,
         cursor: 'pointer',
-        background: gradientColorTheme === 'surface' ? backgroundSurfaceColor : backgroundColor,
+        background: isDarkTheme
+          ? isSurfaceColorTheme
```

Comment:
Hmm, colors are basically flipped for the other theme? 
In any case it looks like `isSurfaceColorTheme` is checked for both cases. By moving it to the outer scope this could probably be simplified.